code/modules/organs/subtypes/augment.dm

```diff
@@ -208,6 +208,7 @@
 	activable = TRUE
 	organ_tag = BP_AUG_EYE_SENSORS
 	action_button_name = "Toggle Eye Sensors"
+	var/active_hud = "disabled"
```

Comment:
I meant we can make hud_types be lowercase. We don't need this active_hud var.

Reply:
active_hud is a bug_fix, as if you select security without security access, that is still the selected_hud, meaning one gets sec records without the HUD.

Reply:
I guess that is one way to fix that. Okay.